### ISSUE TITLE
pathfinder (by class) - Feature

### ISSUE THREAD

**Issue by maddust** [opendatacam/opendatacam#532]
hi, i have seen this feature in other (non-open source) object detection projects and it would be nice to be able to select the pathfinder to track by classes as an option. i see some very useful use cases about this feature such as : 

origin and destination
Retail costumers behavior or (hot zones)
Pedestrian and bike safety analysis  or any trajectories analysis by a specific class 

the pathfinder as is right now is a very useful tool to visualize ID changes and to place count lines in ideal spots within the video. Don't know how hard it will be to implement this as an option in the config file, just a suggestion 

Examples below:

Yellow = pedestrians
Red = Bikes 

<img src="https://i.ibb.co/xSMm3D7/pathfinder-by-class.jpg" alt="pathfinder-by-class" border="0">
<img src="https://i.ibb.co/znSs9CP/heatmap.jpg" alt="heatmap" border="0">

**Comment by vsaw** [comment]
Makes sense. Probaly not going to fit this into v4 but I'll keep it around, maybe it can make it into v5 ;-)

Thanks for the suggestion and apologies for the slow response.
